AGPs/AGP-1.md

```diff
@@ -97,19 +97,19 @@ Below is a list of AGP templates for each track. Copy the template for the track
 To make a Meta track change, you must:
 
 1. Create and submit a pull request changing either AGP-0 or AGP-1.
-2. In a _separate_ pull request, create a new file in the AGPs folder of the AGPs repo.
+2. In a _separate_ draft pull request, create a new file in the AGPs folder of the AGPs repo.
 3. Add a link to the pull request created in step 1 to the new file created in step 2.
 4. Submit the new file from step 2 as a pull request to the AGPs repo. This will be the AGP pull request. If the AGP is approved by ANT holders, the pull request created in step 1 will be merged. If the proposal is rejected and withdrawn, the pull request created in step 1 will be closed.
 
 ### Stage IV: Community Review
-After an AGP has been submitted as a draft to the AGPs repo, it must undergo a Community Review period that starts three weeks before the next Aragon Network vote begins and lasts for one week. Draft AGPs must be submitted as a pull request to the AGPs repo before the Community Review period begins to be considered for the next Aragon Network vote. All draft AGPs that have an open pull request at the time the Community Review begins will automatically be moved to Stage IV for consideration.
+After an AGP has been submitted as a draft to the AGPs repo, it must undergo a Community Review period that starts three weeks before the next Aragon Network vote begins and lasts for one week. Draft AGPs must be submitted as a draft pull request to the AGPs repo before the Community Review period begins to be considered for the next Aragon Network vote. All draft AGPs that have an open pull request at the time the Community Review begins will automatically be moved to Stage IV for consideration.
```

Comment:
```suggestion
After an AGP has been submitted as a draft to the AGPs repo, it must undergo a Community Review period that starts three weeks before the next Aragon Network vote begins and lasts for one week. Draft AGPs must be submitted as a draft pull request to the AGPs repo before the Community Review period begins to be considered for the next Aragon Network vote.`*` All draft AGPs that have an open pull request at the time the Community Review begins will automatically be moved to Stage IV for consideration.

`*` If a draft AGP pull request is accidentally submitted as a ready pull request instead of a draft pull request, the author should put [DRAFT] in the title of the pull request.
```